apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore.IntegrationTests/Google.Cloud.Diagnostics.AspNetCore.IntegrationTests.csproj

```diff
@@ -1,8 +1,8 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
These project files are "mostly auto-generated" based on apis.json. Once we've worked out what they need to look like, we'll need to change the generator and apis.json to make sure we can generate the right thing.